2022-round-1/McLyte/nx_version.txt

```diff
@@ -0,0 +1 @@
+2.6.3
```

Comment:
Could you update this to the latest version of networkx, 2.7.1?

Please also do check your python version.